docs/json/radarr/framestor.json

```diff
@@ -10,7 +10,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\bFraMeSToR\\b"
+        "value": "\\bFraMeSToR$"
```

Comment:
needs to be reverted
doesn't match file names, only release names
`Movie.Title-FraMeSToR.mkv` does not match